src/main/java/groovy/lang/ObjectRange.java

```diff
@@ -302,7 +302,7 @@ public int size() {
                 // let's fast calculate the size
                 final BigDecimal fromNum = new BigDecimal(from.toString());
                 final BigDecimal toNum = new BigDecimal(to.toString());
-                final BigInteger sizeNum = toNum.subtract(fromNum).add(new BigDecimal(1.0)).toBigInteger();
+                final BigInteger sizeNum = toNum.subtract(fromNum).add(BigDecimal.valueOf(1.0)).toBigInteger();
```

Comment:
We can probably just use: BigDecimal.ONE these days.

Reply:
Nice catch! Will change to `BigDecimal.ONE`.